web-ui/components/Editor.vue

```diff
@@ -57,10 +70,49 @@ export default {
       },
     };
   },
+  mounted: function () {
+    let windowWidth = window.innerWidth || document.documentElement.clientWidth || document.body.clientWidth;
+    this.percentInPx = windowWidth / 100;
+    window.addEventListener("mousemove", (e) => this.resizeEditor(e));
```

Comment:
One issue with this is mousemove is always firing, even though most of the time we are not going to be dragging the panels. It is better to `addEventListener` for `mousemove` and `mouseup` in the `startDrag` function so the handler is only active when the user clicks to drag.
Then we need a `removeEventListener` in the stop to remove the event listeners. With this we can remove the `isResizing` variable all together.

All this allows us yo get rid of the `beforeDestroy` method and we can remove all the event listeners from the `mounted` method.